core/src/Pos/Core/Common/AddrSpendingData.hs

```diff
@@ -130,9 +130,9 @@ instance Bi AddrType where
 addrSpendingDataToType :: AddrSpendingData -> AddrType
 addrSpendingDataToType =
     \case
-        PubKeyASD {} -> ATPubKey
-        ScriptASD {} -> ATScript
-        RedeemASD {} -> ATRedeem
+        PubKeyASD _ -> ATPubKey
+        ScriptASD _ -> ATScript
+        RedeemASD _ -> ATRedeem
```

Comment:
The rationale here was to avoid having an unnecessary `RecordWildCards` extension in the file. I overlooked the difference between  `{}` and `_` so I will revert!